docs/release/1.5/release-note-1.5.0.md

```diff
@@ -0,0 +1,15 @@
+# Release Note 1.5.0
+
+## Feature Highlights
+
+- `ONE Compiler`
+    - 
+
+- `ONE Runtime`
+    -
+
+## ONE Compiler
+
+## ONE Runtime
+
```

Comment:
#### CPU backend supports more operations
The following operations are supported on CPU backend :

- ArgMax
- Cos
- ExpandDims
- Fill
- Log
- LogicalNot
- LogicalOr
- Mean
- Neg
- Pow
- ReLU
- ReduceAny
- ReduceProd
- Reverse
- Round
- Select
- SquaredDifference
- Tile
- ZerosLike